### PR TITLE
CIP 36 - Mutual credit proposal

### PR DESCRIPTION
**Pull request by dukejones**

### CODE REVIEW

File: CIPs/cip-mutual_credit_protocol.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+created: 2021-01-23
+requires (*optional): <CIP number(s)>
+replaces (*optional): <CIP number(s)>
+license: Apache 2.0
+---
+
+This is the suggested template for new CIPs.
+
+Note that a CIP number will be assigned by an editor. When opening a pull request to submit your CIP, please
+use an abbreviated title in the filename, `cip-draft_title_abbrev.md`.
```
Comment (YazzyYaz) [inline CIPs/cip-mutual_credit_protocol.md:18]: Please delete this section, it seems copied over from the template :)

File: CIPs/cip-mutual_credit_protocol.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Rationale
+There are many discussions about manipulating supply in various ways to maintain the peg of a stablecoin.  However, mutual credit is a simple and direct mechanism wherein the currency is a pure unit of account. Money comes into being as it is needed.  When the system returns to a steady state, the money disappears back into potentiality. No central authority mints or controls issuance of the currency itself.  As a unit of account without the difficulties and frictions of acquiring and using a currency with central issuance, it is ideally situated to become a primary medium of exchange for commerce.
+
+## Implementation
+https://github.com/ReSource-Network/ReSource-contracts/blob/master/contracts/MutualCreditResource.sol
```
Comment (YazzyYaz) [inline CIPs/cip-mutual_credit_protocol.md:36]: I'd like to see the implementation here in this doc, at least the underlying core logic

**Comment by YazzyYaz** [comment]
Thank you for submitting your CIP proposal! We appreciate your contribution to Celo.

**Comment by dukejones** [comment]
Thanks for your comments!  Working on those changes.

**Review by YazzyYaz — APPROVED** [review]

**Pull request merged**